other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,104 @@
+# Launching a Site on Altis
+
+The process for launching a site on Altis varies depending on the type of project. Nonetheless, there are common steps that need to be done no matter the context and this guide will walk you through what needs to be done in preparation for your site launch.
+
+## Planning
+
+Create a launch issue in the project repository. This issue should include:
```

Comment:
IMO, this should be a "we recommend", because we don't necessarily require people to open issues.